.github/workflows/kiota.yml

```diff
@@ -0,0 +1,49 @@
+name: Generate Kiota client
+run-name: Generate Kiota client
+on: [push]
+permissions:
+  contents: write
+jobs:
+  Generate-Kiota-Client:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-dotnet@v4
+        with:
+          dotnet-version: '8.0.x'
+          
+      - uses: actions/checkout@v3
+          
+      - name: Build Project
+        run: dotnet build WebApiKiota
```

Comment:
This needs to be your solution name, so:

`dotnet build SkyHope.LibraryManager`